core/src/main/resources/jenkins/model/GlobalComputerRetentionCheckIntervalConfiguration/config.groovy

```diff
@@ -0,0 +1,7 @@
+package jenkins.model.GlobalComputerRetentionCheckIntervalConfiguration
```

Comment:
Please use Jelly for newly added UI files.

Reply:
Done via 0fefa24987f2ba752eb793a37bf554c0ea03e5d8 and re-tested. I've seen two variants of providing the help content. For variant a) all the content is placed in a regular HTML file in `war/src/main/webapp/help` and for variant b) a stub jelly file refers to an entry in a dedicated resource bundle with a single key. The code for specifying the actual content with variant b) is [not easy to read](https://github.com/jenkinsci/jenkins/blob/9895d9f1b9b64860dd854639a9cc0cfbabe15fd5/core/src/main/resources/hudson/slaves/RetentionStrategy/Demand/help.properties#L1-L0) (e.g. to spot issues with the HTML markup) as IDEs do not recognize the strings as HTML for highlighting, so I went with a).